src/DebuggerActions/DoesNotUnderstandDebugAction.class.st

```diff
@@ -79,6 +74,11 @@ DoesNotUnderstandDebugAction >> executeAction [
 	self debugger selectTopContext
 ]
 
+{ #category : #accessing }
+DoesNotUnderstandDebugAction >> help [
+    ^ 'Create the missing method in the user prompted class and restart the debugger at that location where it can be edited.'
```

Comment:
Will be such long tooltips readable in all cases even on a small displays?

Reply:
This is the same tooltip as before and I found it readable on my laptop. I think this length is ok.